backfila-embedded/src/main/kotlin/app/cash/backfila/embedded/internal/EmbeddedBackfillRun.kt

```diff
@@ -151,9 +151,9 @@ internal class EmbeddedBackfillRun<B : Backfill>(
   }
 
   override fun scanRemaining() {
-    do {
+    while (!finishedScanning()) {
```

Comment:
It's okay that this block may run 0 instead of at least 1 times?

Reply:
Yeah I ran into a bug in a test. running 0 is fine.